master/networking/migrate-pools.md

```diff
@@ -0,0 +1,198 @@
+default-ipv4-ippool   192.168.0.0/16   true   Always     Never       false
+</pre>
+
+When we run `calicoctl get wep --all-namespaces`, we see that a pod is created using the default range (192.168.52.130/32).
+
+<pre>
+NAMESPACE     WORKLOAD                   NODE      NETWORKS            INTERFACE
+kube-system   coredns-6f4fd4bdf-8q7zp   vagrant   192.168.52.130/32   cali800a63073ed
+</pre>
+
+Let’s get started changing this pod to the new IP pool (10.0.0.0/16).
+
+#### Step 1: Add a new IP pool
+
+We add a new **IPPool** with the CIDR range, **10.0.0.0/16**.
+
+<pre>
+cat <<EOF | calicoctl apply -f -
+apiVersion: projectcalico.org/v3
+kind: IPPool
+metadata:
+name: new-pool
+spec:
+cidr: 10.0.0.0/16
+ipipMode: Always
+natOutgoing: true
+EOF
+</pre>
+
+Let’s verify the new IP pool.
+
+`calicoctl get ippool -o wide`
+
+<pre>
+NAME                  CIDR             NAT    IPIPMODE   DISABLED
+default-ipv4-ippool   192.168.0.0/16   true   Always     false
+new-pool              10.0.0.0/16      true   Always     false
+</pre>
+
+#### Step 2: Disable the old IP pool
+
+List the existing IP pool definition.
+
+`calicoctl get ippool -o yaml > pool.yaml`
+
+<pre>
+apiVersion: projectcalico.org/v3
+items:
+- apiVersion: projectcalico.org/v3
+  kind: IPPool
+  metadata:
+    name: default-ipv4-ippool
+  spec:
+    cidr: 192.0.0.0/16
+    ipipMode: Always
+    natOutgoing: true
+- apiVersion: projectcalico.org/v3
+  kind: IPPool
+  metadata:
+    name: new-pool
+  spec:
+    cidr: 10.0.0.0/16
+    ipipMode: Always
+    natOutgoing: true
+</pre>
+
+Edit pool.yaml.
+
+Disable this IP pool by setting: `disabled: true`
+
+<pre>
+apiVersion: projectcalico.org/v3
+kind: IPPool
+metadata:
+  name: default-ipv4-ippool
+spec:
+  cidr: 192.0.0.0/16
+  ipipMode: Always
+  natOutgoing: true
+  disabled: true
+</pre>
+
+Apply the changes. 
+
+Remember, disabling a pool only affects new IP allocations; networking for existing pods is not affected.
+
+`calicoctl apply -f pool.yaml`
+
+Verify the changes.
+
+`calicoctl get ippool -o wide`
+
+<pre>
+NAME                  CIDR             NAT    IPIPMODE   DISABLED
+default-ipv4-ippool   192.168.0.0/16   true   Always     true
+new-pool              10.0.0.0/16      true   Always     false
+</pre>
+
+#### Step 3: Delete pods from the old IP pool
+
+Next, we delete all of the existing pods from the old IP pool. (In our example, **coredns** is our only pod, but you would repeat this command for each pod in the cluster.)
+
+`kubectl delete pod -n kube-system coredns-6f4fd4bdf-8q7zp`
+
+#### Step 4: Verify that new pods get an address from the new IP pool
+
+1. Create a test namespace and nginx pod.  
+   `kubectl create ns ippool-test`
+
+1. Create an nginx pod.   
+   `kubectl run --namespace=ippool-test nginx --replicas=1 --image=nginx`
+
+1. Verify that the new pod gets an IP address from the new range.    
+   `kubectl -n ippool-test get pods -l run=nginx -o wide`
```

Comment:
And then this would swap `run=nginx` to be `app=nginx`

Reply:
Fixed.